package ShopConfig;

message LimitShopItem
{
	required uint32	  id 		= 1;
	required uint32   equip_id  = 2;  //物品id
	required uint32	  price		= 3;  //价格 	
	required uint32	  limit_cnt	= 4;  //限制次数
}

message LimitShopItems
{
	required uint32	  		open_day 		= 1;
	repeated LimitShopItem	items			= 2; 
}

message LimitShop
{
	repeated LimitShopItems	limit_shop	= 1; 
}